src/api/external/publicationChannelApi.ts

```diff
@@ -1,6 +1,6 @@
 import Axios from 'axios';
 
-import { ExternalApi } from '../../utils/constants';
+import { PUBLICATION_CHANNEL_API_URL } from '../../utils/constants';
```

Comment:
hvorfor har vi ikke ExternalApi-enumen lengre?

Reply:
TS taklet ikke at verdiene til enum kom fra `process.env`
```
export enum ExternalApi {
  AUTHORITY_REGISTER_API_URL = process.env.REACT_APP_AUTHORITY_REGISTER_API_URL,
}
```
^klager på at type 'string | undefined' ikke er assignable to ExternalApi
```
export enum ExternalApi {
  AUTHORITY_REGISTER_API_URL = process.env.REACT_APP_AUTHORITY_REGISTER_API_URL || '',
}
```
^klager på at type string ikke er assignable to ExternalApi